pga2d/multivectors.js

```diff
@@ -28,6 +28,29 @@ export class Even {
     return new Odd(this.yo, -this.xo, this.xy, this.scalar);
   }
 
+  // in 2D PGA, the antidual has exactly the same signs as the dual
+  // so we get this function for free!
+  antidual = this.dual;
+
+  vee_even(other) {
+    // a v b = antidual(dual(a) ^ dual(b))
+    const a_dual = this.dual();
+    const b_dual = other.dual();
+    return a_dual.wedge(b_dual).dual();
```

Comment:
oh I should call `antidual()` here to be explicit, even though it is indeed the same function!